sycl/include/CL/sycl/backend.hpp

```diff
@@ -242,7 +242,8 @@ typename std::enable_if<
 template <backend Backend, typename T, int Dimensions = 1,
           typename AllocatorT = buffer_allocator>
 typename std::enable_if<detail::InteropFeatureSupportMap<Backend>::MakeBuffer ==
-                            true,
+                                true &&
+                            Backend != backend::ext_oneapi_level_zero,
```

Comment:
Do you really need to do this BE check even if you are explicitly providing a L0 BE specialization later on?

Reply:
Yes, I need this check, this is how enable_if works, can't remove this unfortunately.